docs/index.rst

```diff
@@ -19,26 +17,26 @@ Key concepts
+Primitives are base level operations that serve as building blocks for many quantum algorithms and applications. The `base primitive interfaces <https://qiskit.org/documentation/apidoc/primitives.html>`__ are defined in Qiskit Terra, and many Qiskit algorithms use the primitives natively. This abstraction allows you to write the same code, using Qiskit algorithms or otherwise, that can run on different quantum hardware or simulators without having to explicitly manage some of the finer details. See `Introduction to primitives <primitives.html>`__ for further details.
+
+There are currently two primitives defined in Qiskit: Estimator and Sampler.
 
 
 **Estimator**
 
-The estimator primitive allows users to efficiently calculate and interpret expectation
-values of quantum operators required for many algorithms. Users specify circuits that
-prepare quantum states and then Pauli-basis observables to measure on those states. The
-estimator can use advanced error mitigation capabilities to improve the accuracy of the
-returned expectation values.
+The estimator primitive allows you to efficiently calculate and interpret expectation values of quantum operators required for many algorithms. You specify circuits that prepare quantum states and then Pauli-basis observables to measure on those states. The estimator can use advanced error mitigation capabilities to improve the accuracy of the returned expectation values.
 
 **Sampler**
 
-This primitive takes a user circuit as input and returns a quasiprobability distribution
-over the measurement outcomes. This generalizes histograms from quantum circuits to allow
-for error mitigation of readout.
+This primitive takes circuits as input and returns a quasi-probability distribution over the measurement outcomes. This generalizes histograms from quantum circuits to allow for error mitigation of readout.
+
+**Error suppression / mitigation**
+
+Errors occur naturally in a computer, and building fault-tolerant quantum computation is our ultimate goal. While we continue to research how to build error-corrected qubits at scale, Qiskit Runtime offers a number of error suppression and mitigation techniques that alleviate the effect of noise.
 
 **Session**
 
-A session is a contract between the user and the Qiskit Runtime service that ensures that a collection of jobs can be grouped and jointly prioritized by the quantum computer’s job scheduler. This eliminates artificial delays caused by other users’ jobs running on the same quantum device during the session time.
+A session is a contract between the user and the Qiskit Runtime service that ensures that a collection of jobs can be grouped and jointly prioritized by the quantum computer’s job scheduler. This eliminates artificial delays caused by other users’ jobs running on the same quantum device during the session. See `Introduction to sessions <sessions.html>`__ for further details.
```

Comment:
Describing sessions as a "contract" isn't really true.  I think this would be clearer as:
> A session allows a collection of jobs to be grouped and jointly scheduled by the Qiskit Runtime service, facilitating iterative use of quantum computers without incurring queuing delays on each iteration.